crates/bevy_app/src/schedule_runner.rs

```diff
@@ -140,14 +140,22 @@ impl Plugin for ScheduleRunnerPlugin {
                             let mut app = Rc::get_mut(&mut rc).unwrap();
                             let delay = tick(&mut app, wait);
                             match delay {
-                                Ok(delay) => {
-                                    set_timeout(f.borrow().as_ref().unwrap(), delay.unwrap_or(asap))
-                                }
+                                Ok(delay) => set_timeout(
+                                    f.borrow()
+                                        .as_ref()
+                                        .expect("No closure was found for set_timeout."),
```

Comment:
this sounds even more confusing, the user has no way to do anything with that `set_timeout `

Reply:
Looking at this one more, it seems like it's an unreachable error unless I'm misunderstanding the code here.